sessions/js-structure/quiz-app/global.css

```
@@ -0,0 +1,66 @@
+:root {
+	--primary: #ffffff;
+	--secondary: #ff4a11;
+	--secondary-light: #fddcd2;
+	--white: #ffffff;
+	--black: #252629;
+	--ff-primary: sans-serif;
+	--ff-serif: serif;
+	--fs-200: 0.6rem;
+	--fs-300: 0.8rem;
+	--fs-400: 1rem;
+	--fs-500: 1.1rem;
+	--fs-600: 1.5rem;
+	--fs-900: 1.75rem;
+}
+
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+h1,
+h2,
+h3 {
+	line-height: 1.2;
+}
+
+body,
+h1,
+h2,
+h3,
+p {
+	margin: 0;
+}
+
+body {
+	line-height: 1.5;
+	font-family: var(--ff-primary);
+	color: var(--black);
+}
+
+button {
+	font-size: var(--fs-400);
+	font-family: var(--ff-primary);
+	color: var(--black);
+}
+
+ul,
+ol {
+	list-style: none;
+	padding: 0;
+	margin: 0;
+}
+
+ul li,
+ol li {
+	list-style: none;
+	padding: 0;
+}
+
+img {
+	max-width: 100%;
```

```suggestion
	max-width: 100%;
```
Max-width can't be higher than 100%, I don't think? I feel like this is redundant.